src/App.jsx

```diff
@@ -1,27 +1,13 @@
 import React from 'react';
-import Button from 'react-bootstrap/Button';
+// import Button from 'react-bootstrap/Button';
 import 'bootstrap/dist/css/bootstrap.min.css';
 import './App.css';
+import AttitudeForm from './components/AttitudeForm/AttitudeForm';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <div className="mb-2">
-          <Button
-            variant="primary"
-            href="https://react-bootstrap.github.io/getting-started/why-react-bootstrap/"
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            Learn more about react-bootstrap here!
-          </Button>{' '}
-        </div>
-        <p>This is a React App!</p>
-      </header>
+      <AttitudeForm />
```

Comment:
It's okay to do add the component to App when you are testing, but make sure you undo your change before you submit a PR.